### PR TITLE
[docs][ingest-processors] Updated Example in Docs of append processor

### PR DESCRIPTION
**Pull request by vinit-chauhan**
## Proposed commit message
The PR here is to update the example in the append processor in libbeat. The example in the existing doc is ambiguous.

## Checklist

- [x] My code follows the style guidelines of this project
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [ ] I have made corresponding change to the default configuration files
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added an entry in `CHANGELOG.next.asciidoc` or `CHANGELOG-developer.next.asciidoc`.

## Author's Checklist

- [ ]

## Related issues

-

### CODE REVIEW

File: libbeat/processors/actions/docs/append.asciidoc [hunk 1]
```diff
@@ -30,26 +30,34 @@ processors:
...
+      values:
         - static-value
         - ""
       ignore_missing: true
       fail_on_error: true
       ignore_empty_values: true
 ------------------------------------------------------------------------------
 
-Copies the values of `concrete.field`, `array.one` from response field to `my-target-field`:
+Copies the values of `concrete.field`, `array.one` response field and the static values from to `target-field`:
```
Comment (kilfoyle) [inline libbeat/processors/actions/docs/append.asciidoc:45]: Is there a word missing after `the static values from`?

**Review by kilfoyle — APPROVED** [review]
LGTM!

**Comment by vinit-chauhan** [comment]
Hey @kilfoyle - Would you please add appropriate backport tags as well? AFAIK, the feature was merged and released for 8.7.

**Review by pierrehilbert — APPROVED** [review]
LGTM

**Comment by kilfoyle** [comment]
> Hey @kilfoyle - Would you please add appropriate backport tags as well? AFAIK, the feature was merged and released for 8.7.

Thanks to Pierre the backport tags are all set now. :-)

**Pull request merged**